test/filterTracking.test.js

```diff
@@ -0,0 +1,80 @@
+import {buildFilterTrackingObj} from '../src/modules/filterTracking/buildFilterTrackingObj';
```

Comment:
Would suggest, adding default values if no parameters are set.

Reply:
Not required